src/python/pants/backend/python/subsystems/debugpy.py

```diff
@@ -27,7 +27,7 @@ class DebugPy(PythonToolBase):
     default_main = EntryPoint("debugpy")
 
     register_interpreter_constraints = True
-    default_interpreter_constraints = ["CPython>=3.7,<3.11"]
+    default_interpreter_constraints = ["CPython>=3.7"]
```

Comment:
This matches the upstream ICs on the package (and the old value was causing a conflicting ICs error)